### PR TITLE
#184752097 Updating user password

### PR DESCRIPTION
**Pull request by frankflej**
## What does this PR do?
It helps a user update a password
## Description of Task to be completed?
If the user might want to change the password in case they want to this PR helps them in changing their password

## How should this be manually tested?

- Clone the repository using git clone [link](https://github.com/atlp-rwanda/e-comm-team-techsmith-bn)

- Install node packages that the application depends on: `npm install`

- Create a .env file in the root directory and add all environment variables required to run the application. The required variables are listed in the `.env.example` file available in root directory of the application.

- Start the application development server: `npm run dev`

- Test the user registration endpoint available on this address: `https://localhost:{PORT}/api/user/updPassword`, PORT being a local port that the application is running on.

- You can use Postman in your test by provinding the `oldPassword`, `newPassword` and `confPassword` in the body

- If `oldPassword` matches the password in the database and the passwords provided are the same then its a successfull update.

## What are the relevant pivotal tracker/Trello stories?
#184752097

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -11,6 +11,7 @@ DB_HOST =
 DB_PORT =
 
 PORT =  
+<<<<<<< HEAD
```
Comment (nshutijonathan) [inline .env.example]: you have pushed conflicting files `<<<<<<< HEAD`

File: .env.example [hunk 2]
```diff
@@ -20,4 +21,7 @@ USER_SECRET =
 // sendgrid api
 SENDGRID_API_KEY = 
 
-SENDGRID_EMAIL =
\ No newline at end of file
+SENDGRID_EMAIL =
+=======
+tokenSecret = 
+>>>>>>>                                                                           feature(Updating): Updating password                                                                                                               -Updating user password                                                                                                                          Delivers #184752097
```
Comment (nshutijonathan) [inline .env.example]: rebase properly `>>>>>>>`

File: database/migrations/20230331113025-create-users.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const { DataTypes } = require('sequelize');
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+<<<<<<< HEAD:database/migrations/20230330000017-create-user.js
```
Comment (nshutijonathan) [inline database/migrations/20230331113025-create-users.js]: Incomplete rebase

**Review by nshutijonathan — CHANGES_REQUESTED** [review]
rebase properly

**Review by mken1205 — APPROVED** [review]

**Review by bergad37 — APPROVED** [review]

**Review by nishimweprince — APPROVED** [review]
Great work also removing the .nyc_output folder into the files that Git is tracking.

**Review by nshutijonathan — APPROVED** [review]
LGTM

**Pull request merged**